mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -53,6 +51,37 @@ abstract class LDAModel private[clustering] extends Saveable {
   /** Vocabulary size (number of terms or terms in the vocabulary) */
   def vocabSize: Int
 
+  /** Dirichlet parameters for document-topic distribution. */
+  protected def docConcentration: Vector
+
+  /**
+   * Concentration parameter (commonly named "alpha") for the prior placed on documents'
+   * distributions over topics ("theta").
+   *
+   * This is the parameter to a Dirichlet distribution.
+   */
+  def getDocConcentration: Vector = this.docConcentration
```

Comment:
Sorry, I should have thought of this before.  To match this API, can you please remove these 2 get*Concentration methods in favor of making the docConcentration, topicConcentration methods public?